code/build.gradle

```diff
@@ -10,7 +10,7 @@ buildscript {
         maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:4.0.1"
+        classpath 'com.android.tools.build:gradle:7.1.2'
```

Comment:
If possible, let's create a separate PR for the gradle upgrade (changes in gradle scripts, greadle-wrapper.properties), just in case other script changes are required or need to be tested.

Reply:
Gradle version reverted back to 4.0.1 for this PR